plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1729,6 +1737,17 @@ def main():
     # Create an AnsibleModule object with argument specifications
     module = AnsibleModule(argument_spec=element_spec, supports_check_mode=False)
     ccc_ise_radius = IseRadiusIntegration(module)
+    dnac_version = ccc_ise_radius.params.get("dnac_version")
```

Comment:
We don't need this. 
We can directly check..
`if self.dnac_version < self.version_2_3_5_6:`